test-network/prometheus-grafana/grafana/provisioning/dashboards/hlf-performances.json

```diff
@@ -0,0 +1,2407 @@
+				{
+				  "id": "displayName",
+				  "value": "fixed-asset-peer0.org2"
+				}
+			  ]
+			},
+			{
+			  "matcher": {
+				"id": "byRegexp",
+				"options": "fixed-asset:.* peer0.org1.example.com:.*"
+			  },
+			  "properties": [
+				{
+				  "id": "displayName",
+				  "value": "fixed-asset-peer0.org1"
+				}
+			  ]
+			}
+		  ]
+		},
+		"gridPos": {
+		  "h": 11,
+		  "w": 8,
+		  "x": 16,
+		  "y": 46
+		},
+		"id": 42,
+		"options": {
+		  "legend": {
+			"calcs": [],
+			"displayMode": "list",
+			"placement": "bottom"
+		  },
+		  "tooltip": {
+			"mode": "single"
+		  }
+		},
+		"targets": [
+		  {
+			"exemplar": true,
+			"expr": "rate(chaincode_shim_requests_completed{chaincode=~\"fixed-asset.*\"}[$interval])",
+			"hide": false,
+			"interval": "0.01m",
+			"legendFormat": "{{chaincode}} {{instance}}",
+			"refId": "A"
+		  }
+		],
+		"title": "Requests Completed",
+		"type": "timeseries"
+	  },
+	  {
+		"collapsed": false,
+		"datasource": null,
+		"gridPos": {
+		  "h": 1,
+		  "w": 24,
+		  "x": 0,
+		  "y": 57
+		},
+		"id": 49,
+		"panels": [],
+		"title": "Endorser Metrics",
+		"type": "row"
+	  },
+	  {
+		"datasource": null,
+		"fieldConfig": {
+		  "defaults": {
+			"color": {
+			  "mode": "palette-classic"
+			},
+			"custom": {
+			  "axisLabel": "",
+			  "axisPlacement": "auto",
+			  "barAlignment": 0,
+			  "drawStyle": "line",
+			  "fillOpacity": 0,
+			  "gradientMode": "none",
+			  "hideFrom": {
+				"legend": false,
+				"tooltip": false,
+				"viz": false
+			  },
+			  "lineInterpolation": "linear",
+			  "lineWidth": 1,
+			  "pointSize": 5,
+			  "scaleDistribution": {
+				"type": "linear"
+			  },
+			  "showPoints": "auto",
+			  "spanNulls": false,
+			  "stacking": {
+				"group": "A",
+				"mode": "none"
+			  },
+			  "thresholdsStyle": {
+				"mode": "off"
+			  }
+			},
+			"mappings": [],
+			"thresholds": {
+			  "mode": "absolute",
+			  "steps": [
+				{
+				  "color": "green",
+				  "value": null
+				},
+				{
+				  "color": "red",
+				  "value": 80
+				}
+			  ]
+			},
+			"unit": "s"
+		  },
+		  "overrides": [
+			{
+			  "matcher": {
+				"id": "byRegexp",
+				"options": "fixed-asset.* peer0.org2.example.com.*"
+			  },
+			  "properties": [
+				{
+				  "id": "displayName",
+				  "value": "fixed-asset-peer0.org2"
+				}
+			  ]
+			},
+			{
+			  "matcher": {
+				"id": "byRegexp",
+				"options": "fixed-asset.* peer0.org1.example.com.*"
+			  },
+			  "properties": [
+				{
+				  "id": "displayName",
+				  "value": "fixed-asset-peer0.org1"
+				}
+			  ]
+			}
+		  ]
+		},
+		"gridPos": {
+		  "h": 11,
+		  "w": 8,
+		  "x": 0,
+		  "y": 58
+		},
+		"id": 50,
+		"options": {
+		  "legend": {
+			"calcs": [],
+			"displayMode": "list",
+			"placement": "bottom"
+		  },
+		  "tooltip": {
+			"mode": "single"
+		  }
+		},
+		"targets": [
+		  {
+			"exemplar": true,
+			"expr": "rate(endorser_proposal_duration_sum{chaincode=~\"fixed-asset.*\"}[$interval])",
```

Comment:
This is a sum output, but I think we would be interested in an average output

Reply:
Reminder for myself: subsitute with `rate(endorser_proposal_duration_sum{chaincode=~"fixed-asset.*"}[$interval]) / on (instance) endorser_proposals_received`